src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/AddNewUserTest.java

```diff
@@ -16,15 +19,43 @@
 @ActiveProfiles("functional")
 public class AddNewUserTest extends AuthorizationFunctionalTest {
 
+    String orgIdentifierResponse = null;
+
+    @Before
+    public void createAndUpdateOrganisation() {
+        Map<String, Object> response = professionalApiClient.createOrganisation();
+        orgIdentifierResponse = (String) response.get("organisationIdentifier");
+        assertThat(orgIdentifierResponse).isNotEmpty();

```

Comment:
already methods present in the superclass to create and update the organisation . Please reuse the method instead of creating the new method

Reply:
Done